src/types.db

```diff
@@ -195,6 +195,7 @@ players                 value:GAUGE:0:1000000
 pools                   value:GAUGE:0:U
 power                   value:GAUGE:U:U
 pressure                value:GAUGE:0:U
+procevent               value:GAUGE:0:1
```

Comment:
This type is not needed to be added for this plugin. It sends only notifications, not metrics.
The same is about other plugins.